buildtest/cli/compilers.py

```diff
@@ -90,15 +95,15 @@ def compiler_test(configuration):
         console.print(table)
 
 
-def compiler_find(args, configuration):
+def compiler_find(configuration, modulepath, detailed, update):
```

Comment:
Can you please make `modulepath`, `detailed` and `update` as optional argument and document them in docstring